README.md

```diff
@@ -30,6 +30,38 @@ kubectl logs -f $(kubectl get pods --namespace default -l "run=cleanup-operator"
 kubectl create -f https://k8s.io/examples/controllers/job.yaml
 ```
 
+### Parameters
+```
+Usage of ./kube-cleanup-operator:
+  -alsologtostderr
+    	log to standard error as well as files
+  -dry-run
+    	Print only, do not delete anything.
+  -keep-failures int
+    	Number of hours to keep faild jobs, -1 - forever (default) 0 - never, >0 number of hours (default -1)
+  -keep-pending int
+    	Number of hours to keep pending jobs, -1 - forever (default) >0 number of hours (default -1)
+  -keep-successful int
+    	Number of hours to keep successful jobs, -1 - forever, 0 - never (default), >0 number of hours
+  -log_backtrace_at value
```

Comment:
alsologtostderr, log_backtrace_at, log_dir, v, logtostderr, stderrthreshold, vmodule are not specific to the operator. They are coming from the underlying libraries and we cannot guarantee that they will be always present. So people should not rely on these argument. Please do not add these fields.

Reply:
I agree they should'n appear on the parameters list but i think people should know about them to debug effectively. 

What about if i add an "optional parameters" or "other parameters" list explicitely saying that they are coming from the underlying libraries and are not supported on this project. 

About what you were saying below, they appear on the parameters section so would be unnecessary to include a specific section. I propose: 

### Parameters 
(... actual parameters ...)

### Optional Parameters 
### Other Parameters
- verbose- 
- logtosderr ...

What do you think?

Reply:
apologies for the late reply.

I guess having these parameters in the some "other" sections with clear indication that their presence is not guarantied would be fine.